proto/action.proto

```diff
@@ -173,6 +173,12 @@ message ActionPb {
         PlumFinalizeExit plumFinalizeExit = 27;
         PlumSettleDepositPb plumSettleDeposit = 28;
         PlumTransferPb plumTransfer = 29;
+
+        // Block producer protocol actions
+        DonateToProducerFund donateToProducerFund = 30;
```

Comment:
How about?

        DepositToRewardPool
        ClaimFromRewardPool
        SetReward
        GrantReward

Reply:
For DepositToFund and ClaimFromFund, I prefer to have rewarding in the name because all actions are under same package. An detailed name is easier fo an action to be distinguished from others.